package-lock.json

```diff
@@ -0,0 +1,8952 @@
+{
```

Comment:
We use `yarn`. Please remove this file too

Reply:
Why not add a `.npmrc` with `package-lock=false` if you don't want this lock file?

Reply:
As said in the previous comment, we use `yarn`